can_tools/scrapers/__init__.py

```diff
@@ -7,6 +7,9 @@
     CaliforniaTesting,
     CDCCovidDataTracker,
     CDCStateVaccine,
+    CTCountyDeathHospitalizations,
```

Comment:
For some reason, these scrapers never made it into the `__init__` file :man_shrugging: